src/lib/components/Swap/TokenInput.tsx

```diff
@@ -9,6 +10,10 @@ import { DecimalInput } from '../Input'
 import Row from '../Row'
 import TokenSelect from '../TokenSelect'
 
+const LoadingH2 = styled(ThemedText.H2)<{ $loading: boolean }>`
```

Comment:
Instead of a new `LoadingH2` element, this can be wrapped into the existing `ValueInput`.

Just update `ValueInput` to take `$loading`, and plop the css into there.